cmds/core/yes/yes.go

```diff
@@ -7,19 +7,35 @@ package main
 import (
 	"flag"
 	"fmt"
+	"io"
+	"log"
+	"os"
 	"strings"
 )
 
-func main() {
-	flag.Parse()
-	args := flag.Args()
+func runYes(w io.Writer, count uint64, args []string) error {
 	yes := "y"
 	if len(args) > 0 {
```

Comment:
if len(args) can be 0, and that is not an error, could args be args ...string?

Reply:
done